packages/ui/src/components/organisms/RatingSummary/styles.scss

```diff
@@ -0,0 +1,121 @@
+[data-fs-rating-summary] {
+  // --------------------------------------------------------
+  // Design Tokens for Rating Summary
+  // --------------------------------------------------------
+
+  --fs-rating-summary-width                               : 30%;
```

Comment:
It should be 100% in mobile.
<img width="454" alt="image" src="https://github.com/user-attachments/assets/a0354923-6240-444b-b427-986174bbc355" />

My suggestion would be to leave the width as 100%. And we can set this component width accordingly to the screen when styling the section. 
<img width="773" alt="image" src="https://github.com/user-attachments/assets/5da98456-c7ee-4f5a-881c-9d6f1fa36300" />